README.md

```diff
@@ -54,12 +54,14 @@ With Standard Containers we can put an end to that embarrassment, by making INDU
 
 # Contributing
 
-Development happens on github for the spec.  Issues are used for bugs and actionable items and longer
-discussions can happen on the mailing list.  You can subscribe and join the mailing list on
-[google groups](https://groups.google.com/a/opencontainers.org/forum/#!forum/dev).
+Development happens on github for the spec.
+Issues are used for bugs and actionable items and longer discussions can happen on the [mailing list](#mailing-list).
 
-The specification and code is licensed under the Apache 2.0 license found in 
-the `LICENSE` file of this repository.  
+The specification and code is licensed under the Apache 2.0 license found in the `LICENSE` file of this repository.  
+
+## Mailing List
```

Comment:
I'd put the mailing-list section next to the IRC section.  How about sorting channels to the front and usage to the back:

* Mailing List
* IRC
* Discuss your design
* Weekly Call

It would also be nice to standardize around “Title Case” or “Sentance case”, headers, since the current spec isn't very consistent there.